CMakeLists.txt

```diff
@@ -163,13 +163,14 @@ endif()
 # =============================================================================
 # Setup Sphinx documentation
 # =============================================================================
-add_custom_target(
-        prepare_docs
-        COMMAND bash ${PROJECT_SOURCE_DIR}/docs/.prepare_docs.sh
-        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/docs
-)
 find_package(Sphinx QUIET)
 if(SPHINX_FOUND)
+  add_custom_target(
+          prepare_docs
+          COMMAND bash ${PROJECT_SOURCE_DIR}/docs/.prepare_docs.sh
```

Comment:
:) it took me a while to figure out that ```make sphinx'``` does not require ```make prepare_docs```. I was misled by the sphinx output line: ```preparing documents... done```; and  wondered why I couldn't find the dependency.